codes/ArtifactState.ttl

```diff
@@ -0,0 +1,37 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+@prefix wgs84: <https://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
+@prefix time: <https://www.w3.org/TR/owl-time/#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix ids: <https://w3id.org/idsa/core/> .
+@prefix idsc: <https://w3id.org/idsa/code/> .
+@prefix idsm: <https://w3id.org/idsa/metamodel/> .
+
+
+# Instances
+# ---------
+
+idsc:ANONYMIZED a ids:ArtifactState;
+    rdfs:label "is anonymized"@en ;
+    rdfs:comment "The artifact is anonymized so the data source can not be recognized by any further user."@en ;
+.
+
+idsc:PSEUDO_ANONYMIZED a ids:ArtifactState;
```

Comment:
I have never heard "pseudo-anonymized" before. Suggestion: Change to "PSEUDONYMIZED"